lib/mutant/mutator/node/send.rb

```diff
@@ -117,6 +118,22 @@ def emit_receiver_selector_mutations
             .each(&method(:emit_selector))
         end
 
+        # Emit mutation for `#dig`
+        #
+        # - Mutates `foo.dig(a, b)` to `foo.fetch(a).dig(b)`
+        # - Mutates `foo.dig(a)` to `foo.fetch(a)`
+        #
+        # @return [undefined]
+        def emit_dig_mutation
+          return unless selector.equal?(:dig) && arguments.any?
```

Comment:
I wounder if for composed conditions `unless` is the best choice?

`return if !selector.equal? || arguments.none?`

might be easier to read.